### ISSUE TITLE
Swipe to hide

### ISSUE THREAD

**Issue by kaldak** [talklittle/reddit-is-fun#89]
Hi, 
Great app, it's much appreciated.

Maybe it's just me, but I tend to hide any article that i look at but don't up or down vote to keep my frontpage fresh.  It would be very useful if there was a quicker way to hide items.  I think a swipe from left to right across the title would do the job quite well.

Actually, there's probably other functionality that the swipes could be used for as well.  A right to left swipe could save the article.  Alternatively the two swipes could be used to quickly vote on the article.

Thanks!

**Comment by codyro** [comment]
Gestures in general may be a neat feature.. perhaps a bit over zealous. I like the idea of using swipes for hiding articles.

Something similar for comment views as well maybe - I tend to hide the parent as I read through comments.

**Comment by amardeepsingh** [comment]
agreed.  Newsrob has this feature, and it makes the app so much more usable for me.

**Comment by DesiQuintans** [comment]
The lack of this feature is what is stopping me from using this app.

**Comment by atcol** [comment]
Agreed. Fantastic app., but the swipe-to-hide is rapidly becoming a UI convention in apps. It would greatly enhance the user experience. 

I too regularly hide posts due to no vote, so swiping would make this much easier.
